contrib/gitannex/git-annex-remote-rclone-goyle/main.go

```diff
@@ -0,0 +1,93 @@
+// # What even is this?
+//
+// The git-annex-remote-rclone-goyle command ("garrgoyle" for short) enables
+// [git-annex] to store and retrieve content from an rclone remote. It is meant
+// to be run by git-annex rather than directly by users. It speaks the git-annex
+// special remote protocol over stdin and stdout. For more information, check
+// out the parent package: [pkg/github.com/rclone/rclone/contrib/gitannex].
+//
+// # How do I use it?
+//
+// First, install the garrgoyle command:
+//
+//	$ go install github.com/rclone/rclone/contrib/gitannex/git-annex-remote-rclone-goyle
+//
+// Now, run the command with `-h` and follow its instructions:
+//
+//	$ git-annex-remote-rclone-goyle -h
+//
+// [git-annex]: https://git-annex.branchable.com/
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	_ "github.com/rclone/rclone/backend/all"
+	"github.com/rclone/rclone/contrib/gitannex"
+	"github.com/rclone/rclone/fs/config/configfile"
+)
+
+func main() {
+	configfile.Install()
+
+	if err := rootCommand.Execute(); err != nil {
+		panic(err)
+	}
+}
+
+var rootCommand = &cobra.Command{
+	Use:   "git-annex-remote-rclone-goyle",
+	Short: `The garrgoyle command.`,
+	Long: `
+git-annex-remote-rclone-goyle ("garrgoyle" for short) enables git-annex to store
+and retrieve content from an rclone remote. It is an "external special remote
+program" as defined by git-annex's documentation. As such, it is not meant to be
+invoked by users directly.
+
+To set up garrgoyle:
+
+1. Add a new external remote to your git-annex repo.
+
+   The new remote's type should be "rclone-goyle". If it is on your PATH,
+   git-annex will find the git-annex-remote-rclone-goyle command.
+
+   The following example creates a new git-annex remote named "MyRemote" that
+   will use the rclone remote named "SomeRcloneRemote". This rclone remote must
+   be configured in your rclone.conf file, wherever that is located on your
+   system. The rcloneprefix value ensures that content is only written into the
+   rclone remote underneath the "git-annex-content" directory.
+
+    $ git annex initremote MyRemote             \
+        type=external                           \
+        externaltype=rclone-goyle               \
+        encryption=none                         \
+        rcloneremotename=SomeRcloneRemote       \
+        rcloneprefix=git-annex-content
+
+2. Before you trust the garrgoyle command with your precious data, be sure to
+   ***test the remote***. This command is very new and has not been tested on
+   many rclone backends. Caveat emptor!
+
+    $ git annex testremote my-rclone-remote
+
+Happy annexing!
```

Comment:
I should mention that I followed my own instructions and I'm already using the command to back up my own content :)